src/App/Service/Slack.php

```diff
@@ -14,7 +14,6 @@ class Slack
         'eternoendless' => '<@U5GRYPEUC>',
         'jolelievre' => '<@UC4KB9BJS>',
         'kpodemski' => 'kpodemski',
-        'matks' => '<@UB61HUD2A>',
         'matthieu-rolland' => '<@UKW0VAT8S>',
         'NeOMakinG' => '<@UQNF13DAR>',
         'Progi1984' => '<@UL16KUPC5>',
```

Comment:
```suggestion
```

Reply:
Done in https://github.com/PrestaShop/presthubot/pull/52/commits/f5bea383f4c9e0f4829104fc64ae82f82735ca69